### PR TITLE
docker installation instruction updates 

### PR DESCRIPTION
**Pull request by conceptualshark**
### Updates
* _clarity of language in the docker.md introduction page, additional reference links_
* _unified voice and reduced redundancy in docker-compose instructions_
* _proposed page name change for clarity/ease of use: `Running Fidesctl in Docker` -> `Running Fidesctl with Docker Compose`_

### CODE REVIEW

File: docs/fides/docs/quickstart/docker.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Running Fidesctl in Docker
+# Running Fidesctl with Docker Compose
```
Comment (iamkelllly) [inline docs/fides/docs/quickstart/docker.md:1]: On the title selection, can you explain why you decided "with docker compose" vs "in docker"?

File: docs/fides/docs/quickstart/docker.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-The recommended way to get Fidesctl running is via Docker. The following guide will describe how to get things going, step-by-step.
+The recommended method for running Fidesctl is via a Docker Compose environment. 
 
 ## System Requirements
 
-Docker and Docker-Compose are the only requirements here.
+Docker and Docker-Compose are all that is needed to get started.
 
 1. Install `docker` locally (see [Docker Desktop](https://www.docker.com/products/docker-desktop) or your preferred installation). The minimum verified Docker version is `20.10.8`
 1. If your `docker` installation did not include `docker-compose`, make sure to get at least version `1.29.0`. Installation instructions can be found [here](https://docs.docker.com/compose/install/).
```
Comment (iamkelllly) [inline docs/fides/docs/quickstart/docker.md]: how might i know that i don't have docker compose?

Comment (iamkelllly) [reply docs/fides/docs/quickstart/docker.md]: additionally, was there ambiguity to the docker setup instructions below? for example, where do I put the `docker-compose.yml` file?

**Review by iamkelllly — COMMENTED** [review]
@conceptualshark good updates, especially appreciated that you updated both files. i left a few questions for you here! no need to correct the PR if you don't feel it's necessary, really just want to understand your opinion.

File: docs/fides/docs/installation/docker.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
-The fidesctl community releases Docker Images which are reference images for fidesctl. Every time a new version of fidesctl is released, the images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags). There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch.
+Each time a new version of fidesctl is released, the updated images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags). There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch.
 
-These reference images contain all of the extras and dependencies for running the Python application. However they do not contain the required Postgres database.
+These reference images contain all of the extras and dependencies for running the Python application, but do not contain the required Postgres database, covered in this guide's [next steps](../installation/database/).
```
Comment (iamkelllly) [inline docs/fides/docs/installation/docker.md:7]: does this work? I just ran the docs server on my local and this link goes to: `http://0.0.0.0:8000/fides/installation/installation/database/`

File: docs/fides/docs/quickstart/docker.md (hunk 1 above)
Comment (conceptualshark) [reply docs/fides/docs/quickstart/docker.md:1]: Providing the information that Docker Compose is a requirement early on might help prevent a reader from assuming they only needed a docker install, and skipping down to the code samples/content without reading further.

File: docs/fides/docs/quickstart/docker.md (hunk 2 above)
Comment (conceptualshark) [reply docs/fides/docs/quickstart/docker.md]: Definitely agree! Updated based on feedback.

**Comment by conceptualshark** [comment]
I did some additional editing on this pass to cut down the uses of "we," "our," etc, and draw the tone of the installation article more in line with modern style guide recommendations for second-person instructions with clear directions.

**Pull request closed without merging**